src/main/resources/db/migration/V20190812173654__add_user_token.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE users_data ADD COLUMN IF NOT EXISTS token VARCHAR;
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+UPDATE users_data SET token=uuid_generate_v4();
```

Comment:
Where is it required in terms of Phase 1?

Reply:
Also, seems like this extension requires admin permissions to install: let's outline this in Readme (might be a prerequirement since the default OHDSI user is not admin)

Reply:
There is the following command suggested from README:
`create database arachne_portal owner ohdsi`

Does database owner rights enough to create extension?

Reply:
> Where is it required in terms of Phase 1?

One's required at the DataNode side to detect users created in the stand-alone mode.
But it's a good idea to add some verification at the Central side to prevent fraud user to DataNode linkage.

Reply:
Just double-checked using DB owned by `ohdsi` user:
```
sql> CREATE EXTENSION IF NOT EXISTS "uuid-ossp"
[2019-08-14 08:27:12] [42501] ERROR: permission denied to create extension "uuid-ossp"
[2019-08-14 08:27:12] Hint: Must be superuser to create this extension.
```